appimage/build_appimage.sh

```diff
@@ -179,38 +247,108 @@ generate_ssl_cert() {
 }
 
 build_frontend() {
-    pushd "$ISLAND_PATH/cc/ui" || handle_error
-    npm install sass-loader node-sass webpack --save-dev
-    npm update
+  pushd "$ISLAND_PATH/cc/ui" || handle_error
+
+  log_message "Generating front end"
+  npm ci
+  npm run dist
+
+  popd || handle_error
 
-    log_message "Generating front end"
-    npm run dist
-    popd || handle_error
+  remove_node_modules
+}
+
+remove_node_modules() {
+  # Node has served its purpose. We don't need to deliver the node modules with
+  # the AppImage.
+  rm -rf "$ISLAND_PATH"/cc/ui/node_modules
```

Comment:
This will probably be slow. can we use `npx remove-node-modules`?

Reply:
I ran a single test. The `rm` ran in 1.7 seconds. the `npx remode-node-modules` ran in 5.9 seconds.